Dockerfile

```diff
@@ -1,4 +1,25 @@
-FROM golang:1.19
+# Platform is being enforced here to solve issues like:
+# issue /usr/bin/ld: skipping incompatible /usr/local/lib/libavdevice.so when searching for -lavdevice
+#
+# The tools to check the compiled objects format showed:
+# objdump -a /opt/srt_lib/lib/libsrt.so (file format elf64-littleaarch64)
+# objdump -a /usr/local/lib/libavformat.so (file format elf64-little)
+#
+# Once the platform was fixed, the problem disappeared. Even though the configured platform
+# is amd64, the final objects are x64, don't know why yet.
+#
+# FFmpeg/libAV is fixed on version 5.1.2 because go-astiav binding supports it.
+# see https://github.com/asticode/go-astiav/issues/27
+FROM --platform=linux/amd64 jrottenberg/ffmpeg:5.1.2-ubuntu2004  AS base
```

Comment:
Perhaps this is the reason it did not work on my computer `--platform=linux/amd64`? Even running on M1, I'm using Rancher so I'm not sure if the capabilities of both wrappers/engines are the same.

Reply:
I'll try to remove it since we're not using SRT directly anymore, thanks